files/routes/admin/admin.py

```diff
@@ -1426,24 +1443,24 @@ def admin_toggle_ban_domain(v):
 
 @app.post("/admin/nuke_user")
 @limiter.exempt
-@admin_level_required(2)
+@admin_level_required(3)
 def admin_nuke_user(v):
 	user=get_user(request.values.get("user"))
 
 	for post in g.db.query(Submission).filter_by(author_id=user.id).all():
-		if post.is_banned:
+		if post.state_mod != StateMod.Removed:
 			continue
 			
-		post.is_banned = True
-		post.ban_reason = v.username
+		post.state_mod == StateMod.Removed
+		post.state_mod_set_by = v.username
 		g.db.add(post)
 
 	for comment in g.db.query(Comment).filter_by(author_id=user.id).all():
-		if comment.is_banned:
+		if comment.state_mod != StateMod.Removed:
 			continue
 
-		comment.is_banned = True
-		comment.ban_reason = v.username
+		comment.state_mod == StateMod.Removed
```

Comment:
this is a no-op, likely a typo (extra equal sign). this statement doesn't do anything except for compare the two values.

```py
comment.state_mod = StateMod.Removed
```

is what you want

Reply:
Whoops! Good catch.